pkg/ddevapp/ssh_auth_test.go

```diff
@@ -22,6 +23,9 @@ import (
 
 // TestSSHAuth tests basic ssh authentication
 func TestSSHAuth(t *testing.T) {
+	if runtime.GOARCH == "arm64" {
+		t.Skip(("Skipping TestSSHAuth on arm64 because the test-ssh-server is not available for arm64"))
+	}
```

Comment:
I'm sure we can build a test-ssh-server that would work, but I don't think it's worth bothering at this point. However, manual testing shows that `ddev auth ssh` and related capabilities work fine on arm64.

Reply:
Fixed this, added new test-ssh-server and removed this code.